python/pyspark/sql/connect/proto/base_pb2.py

```diff
@@ -28,26 +28,20 @@
 _sym_db = _symbol_database.Default()
 
 
-from pyspark.sql.connect.proto import (
-    commands_pb2 as spark_dot_connect_dot_commands__pb2,
-)
-from pyspark.sql.connect.proto import (
-    relations_pb2 as spark_dot_connect_dot_relations__pb2,
-)
+from pyspark.sql.connect.proto import commands_pb2 as spark_dot_connect_dot_commands__pb2
```

Comment:
Could you tell me how to regenerate these python files?

Reply:
Actually yeah. How do we generate this, @grundprinzip ?

Reply:
I think this is the script that was checked in: https://github.com/apache/spark/blob/master/connector/connect/dev/generate_protos.sh

Reply:
Hopefully we can figure out a better way in the future than manually generating protos for updates.

Reply:
In contrast to Java in Python the generated code is the final artifact. The only way to achieve this would be to use a proper build system for Python but that's probably going to be a much bigger task.

Reply:
> I think this is the script that was checked in: https://github.com/apache/spark/blob/master/connector/connect/dev/generate_protos.sh

Thank you.